packages/eslint-config-ponder/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "eslint-config-ponder",
+  "version": "0.0.81",
```

Comment:
Should make this equal to whatever the latest version is on main (for @ponder/core and create-ponder).